llvm/lib/Target/WebAssembly/WebAssemblyCFGStackify.cpp

```diff
@@ -293,7 +294,7 @@ void WebAssemblyCFGStackify::placeBlockMarker(MachineBasicBlock &MBB) {
     }
   }
 
-  // Decide where in Header to put the BLOCK.
+  // Decide where in MBB to put the BLOCK.
```

Comment:
The relevant variable's name is not `Header` but `MBB`. There are several more fixes on this below.